lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -901,6 +910,11 @@ ParseResult FIRParser::parseType(FIRRTLType &result, const Twine &message) {
     result = baseType.getConstType(true);
     return success();
   }
+
```

Comment:
Probably shouldn't be moving code unrelated to the PR? Looks like rebase artifact.